freeipa/src/main/java/com/sequenceiq/freeipa/flow/freeipa/verticalscale/event/FreeIpaVerticalScaleRequest.java

```diff
@@ -0,0 +1,45 @@
+package com.sequenceiq.freeipa.flow.freeipa.verticalscale.event;
+
+import java.util.List;
+import java.util.StringJoiner;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.sequenceiq.cloudbreak.cloud.context.CloudContext;
+import com.sequenceiq.cloudbreak.cloud.event.resource.CloudStackRequest;
+import com.sequenceiq.cloudbreak.cloud.model.CloudCredential;
+import com.sequenceiq.cloudbreak.cloud.model.CloudResource;
+import com.sequenceiq.cloudbreak.cloud.model.CloudStack;
+
+public class FreeIpaVerticalScaleRequest<T> extends CloudStackRequest<T> {
+
+    private final List<CloudResource> resourceList;
+
+    private final com.sequenceiq.freeipa.api.v1.freeipa.stack.model.scale.FreeIpaVerticalScaleRequest freeIPAVerticalScaleRequest;
```

Comment:
I think it would be nice if you could rename one of the event to avoid this long references